src/packages/DAO/elections.dao.ts

```diff
@@ -53,6 +53,23 @@ export async function startElection(formData: FormData) {
   revalidatePath("/");
 }
 
+const VALUES = ["NOT_STARTED", "ONGOING", "FINISHED"] as const;
+
+export async function addElection(formData: FormData) {
+  const electionName = String(formData.get("electionName"));
```

Comment:
As we're now using zod, you don't need to rely on general type casting, zod should handle it

Reply:
See that, after zod validation the resutls are already in the types we want

![Captura de Tela 2024-03-20 às 19 27 17](https://github.com/podcodar/voting-system/assets/14007745/8116409c-35b7-4eeb-b676-e3d0934b1afc)

Reply:
ok, I understand what you say here.